cmd/zbpack/main.go

```diff
@@ -49,3 +48,30 @@ func main() {
 		panic(err)
 	}
 }
+
+// validate function performs some necessary checks before starting.
+func validate() {
+	if len(os.Args) < 2 {
+		println("Usage: zbpack <directory to analyse or build>")
+		os.Exit(0)
+	}
+
+	c, err := client.NewClientWithOpts(client.WithAPIVersionNegotiation())
+	if err != nil {
+		println("Failed to create Docker client: " + err.Error())
+		os.Exit(0)
+	}
+
+	_, err = c.Ping(context.Background())
+	if err != nil {
+		println("Important: Please make sure the Docker daemon is running.")
+		os.Exit(0)
+	}
+
+	defer func(c *client.Client) {
```

Comment:
Wondering if we can reuse this client instead of just closing it.

Reply:
Of course, We can define a globally unique variable. However, currently there is no other function where this client can be used. I think it may not just closing it in the future.

Reply:
Besides, we use Podman in backend; therefore, this API may failed. cc @yuaanlin

Reply:
This check only in CLI starting. I guess zeabur backend will use pkg/zeaburpack directly :)